gslib/commands/cp.py

```diff
@@ -1107,8 +1107,12 @@ def RunCommand(self):
         copy_helper_opts.daisy_chain,
     )
 
+    process_count, thread_count = self._GetProcessAndThreadCount(
+        process_count=None,
+        thread_count=None,
+        parallel_operations_override=None)
     copy_helper.TriggerReauthForDestinationProviderIfNecessary(
-        dst_url, self.gsutil_api, self.parallel_operations)
+        dst_url, self.gsutil_api, process_count * thread_count)
```

Comment:
Why not just `process_count`?. Wouldn't a reauth get surfaced correctly for single process, multi thread?